tests/unit/test_configuration.py

```diff
@@ -179,14 +178,12 @@ def test_loading_from_yaml(tmp_path: Path) -> None:
     with open(filepath, "w") as file:
         yaml.dump(overrides, file)
 
-    default_config: LayeredConfigTree = get_configuration()[DATASET_SCHEMAS.census.name][
-        Keys.COLUMN_NOISE
-    ][COLUMNS.age.name][NOISE_TYPES.misreport_age.name]
-    default_config = default_config.to_dict()
-    updated_config: LayeredConfigTree = get_configuration(filepath)[
+    default_config: LayeredConfigTree = get_configuration().to_dict()[
```

Comment:
If you're calling `to_dict()` you're probably doing something wrong. See if you can update these tests so that you don't need to use it. If you don't see an obvious way to do so, let's talk tomorrow.